scripts/test_e2e.sh

```diff
@@ -46,7 +46,11 @@ DATASOURCE="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
 export PYTHONPATH=$GOPATH/src/github.com/sql-machine-learning/sqlflow/sql/python
 
 sqlflowserver --datasource=${DATASOURCE} &
-# e2e test for standar SQL
+# e2e test for standard SQL
 SQLFLOW_SERVER=localhost:50051 ipython sql/python/test_magic.py
 # e2e test for xgboost train and prediciton SQL.
 SQLFLOW_SERVER=localhost:50051 ipython sql/python/test_magic_xgboost.py
+# TODO(terrytangyuan): Enable this when ElasticDL is open sourced
+# e2e test for ElasticDL SQL
+# export SQLFLOW_submitter=elasticdl
```

Comment:
A question here: this doesn't quite work yet. Somehow even though I have `SQLFLOW_submitter=elasticdl` here it still uses TensorFlow as the submitter. During my local testing, I had to hard-code executor to use elasticdl submitter to get this working.

Reply:
One possible reason: the `sqlflowserver` is started on line 48, when `SQLFLOW_submitter` is still `""`

Reply:
I see. I’ll update that later when I start uncommenting these lines after open source.